GoogleUtilities/Tests/Unit/Logger/GULLoggerTest.m

```diff
@@ -143,5 +140,64 @@ - (void)testGULResetLogger_ResetsLoggerLevel {
   XCTAssertEqual(loggerLevel, GULLoggerLevelNotice);
 }
 
+#pragma mark - Deprecated Functions
```

Comment:
I thought we determined that we didn't need these?

Reply:
Resolving since we decided to go with the non-breaking approach (deprecating the old functions).